### PR TITLE
Create Enumerable

### PR DESCRIPTION
**Pull request by Chrissiku**
## Project requirements

- [x] Check offical documentation about the following 3 methods in [Enumerable](https://ruby-doc.org/core-3.0.0/Enumerable.html). Make sure that you understand what they are doing.
- [x] Create a class `MyList` that has an instance variable `@list`.
- [x] In `MyList` implement a method `#each` that yields successive members of `@list` and uses the `MyEnumerable` module.
- [x] Create a module `MyEnumerable` that implements the following methods (they should have the same functionality as methods in [Enumerable](https://ruby-doc.org/core-3.0.0/Enumerable.html)):
  - `#all?`
  - `#any?`
  - `#filter`
- [x] Each class and module should have a separate .rb file.
- [x] Verify your solution:
```ruby
# Create our list
irb> list = MyList.new(1, 2, 3, 4)
=> #<MyList: @list=[1, 2, 3, 4]>

# Test #all?
irb> list.all? {|e| e < 5}
=> true
irb> list.all? {|e| e > 5}
=> false

# Test #any?
irb> list.any? {|e| e == 2}
=> true
irb> list.any? {|e| e == 5}
=> false

# Test #filter
irb> list.filter {|e| e.even?}
=> [2, 4]
```

### CODE REVIEW

File: class.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require_relative 'module'
+class Mylist
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+end
```
Comment (Danieldotcomcoder) [inline class.rb]: Inside Mylist class you should implement #each method that yields all members of @list in succession as mentioned in the project requirements , Kindly fix this issue 👍

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @Chrissiku  and @jheart-vic , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: class.rb (hunk 1 above)
Comment (Chrissiku) [reply class.rb]: Thank @Danieldotcomcoder

**Review by whiz25 — APPROVED** [review]
Hi @Chrissiku, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**